pgcli/packages/parseutils.py

```diff
@@ -162,7 +186,7 @@ def find_prev_keyword(sql):
 
     parsed = sqlparse.parse(sql)[0]
     for t in reversed(list(parsed.flatten())):
-        if t.is_keyword or t.value == '(':
+        if is_free_keyword(t) or t.value == '(':
             idx = parsed.token_index(t)
```

Comment:
Yup. You're right. My bad. I'll get that merged in soon. 

@drocco007 Can you try out #224. It looks like the crash is fixed in that branch. Which obviates this additional check for is_free_keyword(). 

But I like to keep the tests.